packages/clipboard/src/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Observable, from, of } from 'rxjs';
+import * as clipboardpoly from 'clipboard-polyfill';
```

Comment:
This is a dependency, should be there on `package.json` file

Reply:
it's there in the `package.json` file of clipboard module.

Reply:
Ok, sorry